src/theme/widget.rs

```diff
@@ -88,3 +88,45 @@ where
         })),
     )
 }
+
+/// A simple button with text and an action defined as an [`Observer`].
+pub fn button_small<E, B, M, I>(text: impl Into<String>, action: I) -> impl Bundle
```

Comment:
Largely copy-pasted from `button`, dunno if that's considered bad form in UI code or not. I imagine this is better than introducing de-duplication functions, as you may want to fine-tune all components in this hierarchy.

Reply:
To remove duplication I renamed the `button` from this PR into `button_base` which takes a size and font size argument. Then I have:

```rust
pub fn button<E, B, M, I>(text: impl Into<String>, action: I) -> impl Bundle
where
    E: Event,
    B: Bundle,
    I: IntoObserverSystem<E, B, M> + Sync,
{
    button_base(text, action, Vec2::new(200.0, 65.0), FontSizes::HEADER)
}

pub fn button_small<E, B, M, I>(text: impl Into<String>, action: I) -> impl Bundle
where
    E: Event,
    B: Bundle,
    I: IntoObserverSystem<E, B, M> + Sync,
{
    button_base(text, action, Vec2::new(180.0, 35.0), FontSizes::UI)
}

```

For my needs at least this is fine, font size and size are the main things that change for these kinds of "standard" buttons.

Reply:
That's neat, I'll copy that design :)